Azure-Samples/chat-with-your-data-solution-accelerator
# force solution to build for amd64 architecture as arm64 does not supp…

…ort azure functions

## Purpose

* It makes it possible for this repo to run on Macs (Apple Silicon) with Rosetta. 
* It forces the devcontainer to build under amd64 architecture - this can run successfully in Macs with Apple Silicon and Rosetta.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
Once you see the message **Reopen in Container**, do click that button
```

## What to Check
Verify that the following are valid
* Container builds successfully in both PCs and Macs with Apple Silicon

## Other Information
closes https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/pull/486
